app/app.go

```diff
@@ -493,6 +502,9 @@ func New(
 		),
 	)
 
+	app.FeeMarketKeeper = feemarketkeeper.NewKeeper(appCodec, keys[feemarkettypes.StoreKey], app.AccountKeeper, &feemarkettypes.TestDenomResolver{}, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+	app.FeeMarketKeeper.SetDenomResolver(&feemarkettypes.TestDenomResolver{}) // TODO
```

Comment:
Looks like this is still pending.

Could you address + resolve?

Reply:
This is handled in the feebs PR.